<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>5003</width>
    <height>3061</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>0</y>
      <width>5161</width>
      <height>2911</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/res/37653.jpg);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
font: italic 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Code de l’article :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>420</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>360</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>360</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
font: italic 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Nom de l’article :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
font: italic 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Catégorie :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
font: italic 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Date d'achat : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
font: italic 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>date d’expiration :</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>210</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
font: italic 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Statut :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>260</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>310</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>360</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
font: italic 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>370</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
font: italic 9pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>etat</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>1031</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QTableView {
            background-color: #1E1E1E;  /* Dark background */
            color: #FFFFFF;              /* White text */
            border: 1px solid #3C3C3C;   /* Border color */
        }
        QHeaderView {
            background-color: #1E1E1E;   /* Header background */
            color: #FFFFFF;              /* Header text color */
            border: 1px solid #3C3C3C;   /* Header border */
        }
        QTableView::item {
            background-color: #1E1E1E;  /* Item background */
            color: #FFFFFF;              /* Item text color */
        }
        QTableView::item:selected {
            background-color: #007ACC;   /* Selected item background */
            color: #FFFFFF;              /* Selected item text color */
        }
    </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>360</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>5003</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
